runtime/src/bank.rs

```diff
@@ -62,8 +62,9 @@ pub struct BankRc {
 }
 
 impl BankRc {
-    pub fn new(account_paths: Option<String>) -> Self {
+    pub fn new(account_paths: Option<String>, id: usize) -> Self {
```

Comment:
what does "id" represent, can you use a type alias or another name so this is easier to read?

Reply:
this corresponds to AppendVecId, will replace to use AppendVecId